localization/english/wc_other_titles_l_english.yml

```diff
@@ -1,173 +1,674 @@
 ﻿l_english:
- k_atalai:0 "Atal'ai"
- 
- d_church_of_the_holy_light:0 "Church of the Holy Light"
- d_church_of_the_holy_light_article:0 "the "
- 
- d_church_scarlet_light:0 "Scarlet Monastery"
- d_church_scarlet_light_article:0 "the "
- 
- d_naaru:0 "Faithful of Naaru"
- d_hand_of_argus:0 "Hand of Argus"
- d_bloodscalp:0 "Bloodscalp"
- d_skullsplitter:0 "Skullsplitter"
- 
+ #########################
+ #		Without land		   #
+ #########################
+ ### Orcs ###
  e_horde:0 "Horde"
  e_horde_adj:0 "Horde"
  e_horde_article:0 "the "
  
- e_empire_of_zul:0 "Empire of Zul"
- e_empire_of_zul_adj:0 "Zulian"
- e_empire_of_zul_article:0 "the "
+ k_orc_blackrock:0 "Blackrock clan"
```

Comment:
All words in title name except `the` should be capitalized.

Reply:
done in 7c837aee4 and e11de47e1